src/components/Missions.js

```diff
@@ -1,17 +1,41 @@
-import { useDispatch } from 'react-redux';
-// import Table from 'react-bootstrap/Table';
+import { useDispatch, useSelector } from 'react-redux';
+import Table from 'react-bootstrap/Table';
 import { useEffect } from 'react';
+import { Badge } from 'react-bootstrap';
 import { fetchMissions } from '../redux/mission/missionSlice';
 
```

Comment:
![image](https://user-images.githubusercontent.com/100031798/227005283-e00cadcc-f157-4237-8dbb-bf0bf38cbc24.png)

Reply:
Hi @josuerushanika 
Thank You for addressing the issues above. It appears that the project that does run in the browser as described . However i would  ask that you install the following **React Bootsrap** dependencies and try running the program again.

If the issues persist afterwards, kinfly draw my attention so i find another solution since it runs without flaws here.

Kindly follow [this link](https://react-bootstrap.github.io/getting-started/introduction) to install React Boostrap